### PR TITLE
Upgrade picomatch so image-sequencer works in Safari

### LINKED ISSUE #1738: Not working in Safari
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?

Load page, can't take or upload photos.
Console error "Invalid regular expression: invalid group specifier name"

Quick google search indicates that error message may be due to using look-behinds in regular expressions.
https://stackoverflow.com/questions/51568821/works-in-chrome-but-breaks-in-safari-invalid-regular-expression-invalid-group

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

jlev

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

Mac OS 10.15.7
Safari 14.0

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by jlev**
for publiclab/image-sequencer#1738

Fixes #1738

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below
* [ ] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by jlev** [comment]
Had one failed test locally, not sure if that's due to building on a Mac...

```

  Failed Tests: There was 1 failure

    convolution module works correctly

      ✖ convolution module works correctly with Gif

```

**Review by jywarren — APPROVED** [review]
Nice!

**Comment by jywarren** [comment]
Looks great, thanks Josh!!!

**Comment by jywarren** [comment]
Feel free to merge this, @publiclab/is-maintainers !

File: package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "opencv.js": "^1.2.1",
     "ora": "^4.0.3",
     "pace": "0.0.4",
+    "picomatch": "^2.2.2",
```
Comment (HarshKhandeparkar) [inline package.json:73]: Why is it added to the package.json? Isn't it a package-lock dependency?

Comment (jywarren) [reply package.json:73]: Maybe to lock it intentionally for Safari...?

Comment (HarshKhandeparkar) [reply package.json:73]: package-lock locks. Also, package.json versions do not interfere with the peer dependency versions...

Comment (jywarren) [reply package.json:73]: Any input on this @jlev? Shall we move it to the `package-lock.json` file instead then?

Comment (jywarren) [reply package.json:73]: @HarshKhandeparkar would you like to try making the suggested changes using comment suggestions? I'll remove it here and you can make another suggestion in `package-lock.json`. Or, would there be any changes required, given that the package.json change triggered the upgrade in package-lock.json?

Comment (jywarren) [reply package.json:73]:
```suggestion
```

Comment (jywarren) [reply package.json:73]: If this suggestion looks sufficient, feel free to commit it!

**Review by jywarren — CHANGES_REQUESTED** [review]

**Comment by HarshKhandeparkar** [comment]
No, just a single suggestion isn't sufficient. We'll have to delete using npm only. We ARE NOT supposed to handle lockfiles and doing anything like that will only lead to disaster. (according to npm ig)

**Comment by jywarren** [comment]
OK, want to try this out using GitPod or otherwise push a commit to this
branch? I'm not that familiar with changing just the lockfile as I usually
let Dependabot handle those.

**Comment by HarshKhandeparkar** [comment]
I can remove the dependency, one sec.
radio voice: *initiating gitpod*
radio: *igniting in t - forever*
(Gitpod really does take forever :joy:)

**Comment by HarshKhandeparkar** [comment]
Gitpod isn't allowing me to push, but `npm uninstall picomatch` does the job. I also checked the diff, apparently there is only one line that was changed in the lockfile(by jlev) and that remains. So just removing that one line from package.json works :)

**Review by HarshKhandeparkar — APPROVED** [review]

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Done! Nice and many thanks to @jlev !!

**Pull request merged**